docs/examples.rst

```diff
@@ -158,16 +158,28 @@ The other important attribute is ``runs``, which is a list of the
 :py:class:`aospy.Run` objects that pertain to simulations performed in
 this particular model.
 
+Lastly, grid attibutes (like time, latitude, longitude, etc.), often have
+different names from model to model.  ``aospy`` can handle most typical grid
+attribute names automatically; however, if your model uses unusual grid
+attribute names you can add a ``grid_attrs`` dictionary to your
+:py:class:`~aospy.Model` object to define a mapping between ``aospy`` internal
+names and names for the grid attributes that can be found in your data files.
+As an example, here only the land mask and longitude variables have atypical
```

Comment:
I think it would help to be even more explicit, e.g. something like "In our example data, the land mask and longitude variables both have non-standard names: 'custom_land_mask' and 'custom_lon', respectively....